main/org.thingpedia.iot.humidity/manifest.tt

```diff
@@ -0,0 +1,13 @@
+abstract class @org.thingpedia.iot.humidity
+#_[thingpedia_name="Humidity Sensor"]
+#_[thingpedia_description="Interface for Humidity Sensor."]
+#_[description="Interface for Humidity Sensor."]
+#[license="BSD-3-Clause"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query humidity(out state : Enum(humid,normal), out value : Number)
+  #_[canonical="humidity sensor state"]
+  #_[confirmation="the humidity from $__device"]
+  #_[formatted=["It is ${state}.", "The humidity level is ${value} percent."]];
```

Comment:
`#_[formatted]` annotations that only contain text should be replaced with `#_[result]` annotations:
```
#_[result=["the humidity level is ${value} percent", "it is ${state} in the room"]]
```

The semantics are different: only one phrase will be shown to the user, rather than both (which one depends by which values are available). We should write a brief but maximally informative phrase. Also, `#_[result]` should be lower-cased and have no punctuation.

Additionally, `humidity` returns two values, the discrete state and the numeric value. It does not make sense for the user to choose one value or the other, especially given that only one value is set at any given time, so we need to prevent any _projection_ on the return value of the function. You do so by setting `#[minimal_projection]` to contain both parameters:
```
#[minimal_projection=["state", "value"]]
```
(minimal_projection is the list of output parameters that are always retrieved from the API, regardless of what the user asks)

Reply:
ok